templates/module/home/img_box.html

```diff
@@ -36,15 +36,15 @@
     <th:block th:replace="~{module/home/focus_tou}"></th:block>
     <div
       class="header-info no-select"
-      th:classappend="${#strings.isEmpty(site.subtitle) ? '' : 'info-desc'}"
-      th:if="${theme.config.mainScreen.focus_infos and (not #strings.isEmpty(site.subtitle) or not #arrays.isEmpty(theme.config.sns.items))}"
+      th:classappend="${#strings.isEmpty(theme.config.mainScreen.focus_info_self_intro_text) ? '' : 'info-desc'}"
+      th:if="${(theme.config.mainScreen.focus_info_self_intro or theme.config.mainScreen.focus_info_socials) and (not #strings.isEmpty(theme.config.mainScreen.focus_info_self_intro_text) or not #arrays.isEmpty(theme.config.sns.items))}"
```

Comment:
```suggestion
      th:classappend="${#strings.isEmpty(theme.config.mainScreen.focus_info_self_intro) ? 'info-desc' : ''}"
      th:if="${(theme.config.mainScreen.focus_info_self_intro or theme.config.mainScreen.focus_info_socials) and (not #arrays.isEmpty(theme.config.sns.items))}"
```

已经增加了选项是否开启的状态下，这里就不需要再判断内容了，否则当内容不为空时，是否开启就没有作用了。